src/main/java/eu/dissco/backend/controller/UserController.java

```diff
@@ -71,6 +74,21 @@ public ResponseEntity<JsonApiWrapper> getUser(Authentication authentication,
     return ResponseEntity.ok(new JsonApiWrapper(response, new JsonApiLinks(getPath(request))));
   }
 
+  @ResponseStatus(HttpStatus.OK)
+  @GetMapping(value = "/mjr")
+  public ResponseEntity<JsonApiListResponseWrapper> getMasJobRecords(
+      @RequestParam(defaultValue = DEFAULT_PAGE_NUM) int pageNumber,
+      @RequestParam(defaultValue = DEFAULT_PAGE_SIZE) int pageSize,
+      @RequestParam(required = false) AnnotationState state,
+      Authentication authentication,
+     HttpServletRequest request) {
+    var userId = getNameFromToken(authentication);
+    log.info("User: {} has requested a list of their Mas Job records", userId);
+    var response = service.getMasJobRecordsForUser(userId, getPath(request), pageNumber, pageSize, state);
```

Comment:
We need to think if a user can request this from other users. Maybe build in a check that the user can only collect her/his own mjr's. We can always loosen authorisation later.

Reply:
Isn't this already what is happening? The userId is derived from the token of whoever made the request, which is directly piped into the function to get MJRs for the user. The only way to get a different user's MJRs would be to use a different token.

Reply:
Oh yeah you are right. :tada: